src/getters.jl

```diff
@@ -1885,4 +1885,90 @@ function GetDatabaseCohorts(
     
 end
 
-export GetDatabasePersonIDs, GetPatientState, GetPatientGender, GetPatientRace, GetPatientAgeGroup, GetPatientVisits, GetMostRecentConditions, GetMostRecentVisit, GetVisitCondition, GetPatientEthnicity, GetDatabaseYearRange, GetVisitPlaceOfService, GetVisitConcept, GetVisitDate, GetDrugExposures, GetDrugConceptIDs, GetDrugAmounts, GetVisitProcedure, GetDatabaseCohorts, GetCohortSubjects, GetCohortSubjectStartDate, GetCohortSubjectEndDate, GetDrugExposureIDs
+"""
+function GetDrugExposureEndDate(drug_exposure_ids; tab = drug_exposure)
+
+Given a list of drug_exposure IDs, find their corresponding drug_exposure_end_date ID.
+
+# Arguments:
+
+- `drug_exposure_ids` - list of `drug_exposure_id`'s; each ID must be of subtype `Float64`
+
+- `conn` - database connection using DBInterface
+
+
+# Keyword Arguments:
+
+- `tab` - the `SQLTable` representing the Drug Exposure table; default `drug_exposure`
+
+# Returns
+
+- `df::DataFrame` - a two column `DataFrame` comprised of columns: `:drug_exposure_id` and `:drug_exposure_end_date`
+"""
+
+function GetDrugExposureEndDate(
+    drug_exposure_ids,
+    conn;
+    tab = drug_exposure 
+)
+
+    df = DBInterface.execute(conn, GetDrugExposureEndDate(drug_exposure_ids; tab=tab)) |> DataFrame
+
+    return df
+end
+
+"""
+function GetDrugExposureEndDate(df:DataFrame, conn; tab = drug_exposure)
+
+Given a `DataFrame` with a `:drug_exposure_id` column and `:drug_exposure_end_date` column, return the `DataFrame` with an associated 
+`:drug_exposure_end_date` corresponding to a given `drug_exposure_id` and `drug_exposure_end_date` in the `DataFrame`
```

Comment:
I'd rewrite this as:

Given a `DataFrame` with a `:drug_exposure_id` column, return the `DataFrame` with an associated 
`:drug_exposure_end_date` corresponding to a given `drug_exposure_id` in the `DataFrame`.